ci.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
-# Please run it from root project directory
+set -xe
```

Comment:
>In both CI configurations you changed too build.sh instead of this ci.sh

@vanniktech that was the sentence I DID not understand because it was "yes, that's true… so what…?" to me, lol, sorry!